ycms/cms/models/room.py

```diff
@@ -51,6 +51,16 @@ def available_beds(self):
         """
         return self.beds.filter(is_available=True).count()
 
+    @cached_property
+    def occupied_beds(self):
+        """
+        Helper property for accessing the rooms occupied bed count
+
+        :return: number of occupied beds in the room
+        :rtype: int
+        """
+        return self.beds.filter(is_available=False).count()
+
```

Comment:
```suggestion
    @cached_property
    def occupied_beds(self):
        """
        Helper property for accessing the rooms occupied bed count

        :return: number of occupied beds in the room
        :rtype: int
        """
        return self.total_beds - self.available_beds

```
Unfortunately, after #40 is merged we will no longer be able to use `is_available` inside `filter()`, since that PR turns the attribute into a property. It's probably easier to already account for this here than change it later 😄